src/Alert/Alert.js

```diff
@@ -0,0 +1,159 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+
+import { InformationCircle, Check, Alert as AlertTriangle, AlertCircle, Close } from "../icons";
+import ButtonLink from "../ButtonLink/ButtonLink";
+
+export const TYPE_OPTIONS = {
+  INFO: "info",
+  SUCCESS: "success",
+  WARNING: "warning",
+  CRITICAL: "critical",
+};
+
+type IconProps = {
+  icon: React.Node,
+  type: string,
+};
+
+const Icon = ({ icon, type }: IconProps) => {
+  // Icon should be boolean and TRUE
+  if (typeof icon === "boolean" && icon) {
+    if (type === TYPE_OPTIONS.INFO) {
+      return <InformationCircle />;
+    }
+    if (type === TYPE_OPTIONS.SUCCESS) {
+      return <Check />;
+    }
+    if (type === TYPE_OPTIONS.WARNING) {
+      return <AlertTriangle />;
+    }
+    if (type === TYPE_OPTIONS.CRITICAL) {
+      return <AlertCircle />;
+    }
+  }
+  return icon;
+};
+
+type Props = {
+  type: $Values<typeof TYPE_OPTIONS>,
+  title?: string,
+  icon?: React.Element<any> | boolean,
+  closable: boolean,
+  onClose: () => void,
+  children: React.Node,
+  theme: typeof defaultTokens,
+};
+
+const StyledAlert = styled.div`
+  position: relative;
+  display: flex;
+  width: 100%;
+  padding: ${({ theme, icon }) =>
+    icon ? `${theme.paddingAlert} ${theme.paddingAlertWithIcon}` : theme.paddingAlert};
+  padding-right: ${({ theme, closable }) => closable && theme.spaceXXLarge};
+  border-radius: ${({ theme }) => theme.borderRadiusNormal};
+  background: ${({ tokens, type }) => tokens.backgroundAlert[type]};
+  color: ${({ tokens, type }) => tokens.colorTextAlert[type]};
+  font-family: ${({ theme }) => theme.fontFamily};
+  font-size: ${({ theme }) => theme.fontSizeTextNormal};
+  box-sizing: border-box;
+`;
+
+const IconContainer = styled.div`
+  flex-shrink: 0;
+  margin-right: ${({ theme }) => theme.spaceSmall};
+  color: ${({ tokens, type }) => tokens.colorIconAlert[type]};
+`;
+
+const Title = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: ${({ theme, hasChildren }) => hasChildren && theme.spaceXSmall};
+  font-weight: ${({ theme }) => theme.fontWeightBold};
+  line-height: ${({ theme }) => theme.lineHeightHeading};
+  min-height: ${({ theme }) => theme.heightIconMedium};
+`;
+
+const Content = styled.div`
+  display: flex;
+  align-items: center;
+  min-height: ${({ theme, title }) => !title && theme.heightIconMedium};
+  margin-bottom: ${({ theme, title }) => title && theme.spaceXXSmall};
+`;
+
+const CloseContainer = styled.div`
+  position: absolute;
```

Comment:
We can use here `flex` as well, right? Personally, I don't like mixing flex with `position`

Reply:
I don't want to affect layout by close icon, I think `absolute` is better

Reply:
OK, as you think, but I rather use only `flex`.
On the **line 90** you have hardcoded `-16px`, it needs to be token.